<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="cover">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>61</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>451</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Camera</string>
     </attribute>
     <widget class="QComboBox" name="comboBoxFun">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>NULL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>detect arrow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>fast moving detect</string>
       </property>
      </item>
     </widget>
     <widget class="QTextEdit" name="serial_output">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>141</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>111</width>
        <height>122</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>S</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>V</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>102</width>
        <height>87</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="serial_list"/>
       </item>
       <item>
        <widget class="QPushButton" name="serial_refresh">
         <property name="text">
          <string>refresh</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="serial_open">
         <property name="text">
          <string>open serial</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Calibrate</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>311</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="1">
            <widget class="QLineEdit" name="editCarX"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="editNumY">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>5</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
             <property name="cursorPosition">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>n_col</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="editCarY"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="editNumX">
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>car_x</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>box_len</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>car_y</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>n_row</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="editBoxLen">
             <property name="text">
              <string>31</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="1">
            <widget class="QLineEdit" name="editWorldY">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>x</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>y</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="editWorldX">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="editWorldZ">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>z</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Inverse axis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkInvX">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkInvY">
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkInvZ">
           <property name="text">
            <string>z</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnCaliStart">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCaliContinue">
           <property name="text">
            <string>Resume</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="1">
            <widget class="QRadioButton" name="rbtnDR">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="rbtnDL">
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="rbtnUL">
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="rbtnUR">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnCaliGen">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Generate</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="info">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>250</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QComboBox" name="comBoxMode">
       <item>
        <property name="text">
         <string>Color</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Depth</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Infrared 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Infrared 2</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Open Camera</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>FPS : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelFps">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelRGB">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelCoor">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>550</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1199</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionOpen_Device"/>
    <addaction name="actionClose_Device"/>
   </widget>
   <widget class="QMenu" name="menuSelect">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionRealsense"/>
    <addaction name="actionZed"/>
    <addaction name="actionUsb"/>
    <addaction name="actionKinect"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionhahahah"/>
    <addaction name="actionheiheiei"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSave_Img"/>
    <addaction name="actionRecord"/>
    <addaction name="actionStop_Record"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionClear"/>
   </widget>
   <addaction name="menuFILE"/>
   <addaction name="menuSelect"/>
   <addaction name="menuTools"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRealsense">
   <property name="text">
    <string>Realsense</string>
   </property>
  </action>
  <action name="actionZed">
   <property name="text">
    <string>Zed</string>
   </property>
  </action>
  <action name="actionUsb">
   <property name="text">
    <string>Usb</string>
   </property>
  </action>
  <action name="actionKinect">
   <property name="text">
    <string>Kinect</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionOpen_Device">
   <property name="text">
    <string>Open Device</string>
   </property>
  </action>
  <action name="actionClose_Device">
   <property name="text">
    <string>Close Device</string>
   </property>
  </action>
  <action name="actionhahahah">
   <property name="text">
    <string>hahahah</string>
   </property>
  </action>
  <action name="actionheiheiei">
   <property name="text">
    <string>heiheiei</string>
   </property>
  </action>
  <action name="actionSave_Img">
   <property name="text">
    <string>Save Img</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="text">
    <string>Start Record</string>
   </property>
  </action>
  <action name="actionStop_Record">
   <property name="text">
    <string>Stop Record</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>editNumX</tabstop>
  <tabstop>editNumY</tabstop>
  <tabstop>editBoxLen</tabstop>
  <tabstop>editWorldX</tabstop>
  <tabstop>editWorldY</tabstop>
  <tabstop>editWorldZ</tabstop>
  <tabstop>editCarX</tabstop>
  <tabstop>editCarY</tabstop>
  <tabstop>btnCaliStart</tabstop>
  <tabstop>btnCaliContinue</tabstop>
  <tabstop>btnCaliGen</tabstop>
  <tabstop>chkInvZ</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>chkInvX</tabstop>
  <tabstop>rbtnDR</tabstop>
  <tabstop>rbtnDL</tabstop>
  <tabstop>rbtnUL</tabstop>
  <tabstop>rbtnUR</tabstop>
  <tabstop>chkInvY</tabstop>
  <tabstop>comBoxMode</tabstop>
  <tabstop>checkBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
